pandas_gbq/gbq.py

```diff
@@ -421,6 +439,17 @@ def sizeof_fmt(num, suffix='B'):
             num /= 1024.0
         return fmt % (num, 'Y', suffix)
 
+    def query_price_for(self, bytes_num):
```

Comment:
Could you also add a unit test for `query_price_for` so that our code coverage doesn't decrease?